torch/_dynamo/utils.py

```diff
@@ -25,7 +25,13 @@
 from functools import lru_cache
 from typing import Any, Dict, List
 
-import numpy as np
+try:
```

Comment:
should this be the only place to assign `np` and `HAS_NUMPY`, and everyone else should import it from utils?

Reply:
That seems reasonable, it is a bit repetitive to do this everywhere.